AGENTS.md

```diff
@@ -3,69 +3,321 @@
 This file provides specific context and instructions for AI coding agents to
 interact effectively with this C# project.
 
-
 ## Project Overview
 
-TestResults is a C# library for creating TRX test result files.
+TestResults is a C# library for creating TRX test result files. The library is designed to be:
 
+- **Lightweight**: Zero external dependencies
+- **Simple**: Easy-to-use API for creating TRX files
+- **Type-Safe**: Strongly-typed C# objects
+- **Cross-Platform**: Supports .NET 8, 9, and 10
 
 ## Technologies and Dependencies
 
-* **Language**: C# 12
-* **.NET Frameworks**: .NET 8, 9, and 10
-* **Primary Dependencies**: None
+### Runtime
+
+- **Language**: C# 12
+- **.NET Frameworks**: .NET 8, 9, and 10
+- **Primary Dependencies**: None (by design)
```

Comment:
Having no dependencies isn't actually by design - it just doesn't have any right now.